cmd/machine-config-daemon/pivot.go

```diff
@@ -36,7 +36,9 @@ const (
 // TODO: fill out the allowlist
 // tuneableArgsAllowlist contains allowed keys for tunable arguments
 var tuneableArgsAllowlist = map[string]bool{
-	"nosmt": true,
+	"nosmt":                              true,
+	"systemd.unified_cgroup_hierarchy=0": true,
+	"mitigations=auto,nosmt":             true,
```

Comment:
Pivot does understand which platform is being run on - maybe to be picky and cleaner we can allow these two only when !rhcos for now and expand it later when okd grows more platform?

Reply:
sgtm!

Reply:
We could reset it to be FCOS specific, but I don't see a reason for that - RHCOS users should be able to modify that parameter if it happens to be set/unset

Reply:
I think this is now https://github.com/openshift/machine-config-operator/pull/1925/files which merged right?

Reply:
oh, right, so its not urgent for OKD then.

Do we want to allow this for RHCOS? I don't think there's need right now so we can keep it close and reopen if necessary

Reply:
I think that means we can close this PR for now? I'll do that now, but feel free to reopen if this merits further discussion!